server/routes/login.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+
+router.post('/api/login', (req, res) => {
+  const { email, password } = req.body;
+  // Here we would typically check the credentials against a database
+  // For simplicity, we're just going to accept any credentials
+  res.sendStatus(200);
```

Comment:
It's better to return a response body with status code, instead of just status code. This will help in better handling on the client side.